capabilities/code-maintainability.md

```diff
@@ -37,77 +56,91 @@ Don't worry if the description doesn't exactly match your situation. The descrip
 
 ### Previously Written Code
 
-1. **Brittle Codebase:** Changing any code is time-consuming, complex, and error prone.
-2. **Fairly Complex:** Most changes require significant refactoring, and it's difficult to predict the impact of changes on the overall system.
-3. **Partially Modular:** Most parts of the system are modular and easy to update, but some are complex and difficult to work with.
-4. **Well Organized:** When changes are made to the existing codebase, they don’t tend to require much rework.
+1. **Brittle Codebase:** Changing any code is time-consuming, complex, and prone to error.
+2. **Fairly Complex Codebase:** Most changes require significant refactoring, and it's difficult to predict the impact of changes on the overall system.
+3. **Partially Modular Codebase:** Most parts of the system are modular and easy to update, but some are complex and difficult to work with.
+4. **Well-organized Codebase:** When changes are made to the existing codebase, they don’t tend to require much rework.
+
+Now average your scores for the two responses and that will be your overall score for this capability.
+
+<ed> By "two responses" do we mean New Code and Previously Written Code? I think we could make this instruction clearer for the reader. My understanding is that the reader is to score themselves on each of the 4 areas in each of the two "subcategories" above (New Code and Previously Written Code). Do they then average the scores of each subcategory or do they average the scores of ALL areas (which would be 8 areas total)? </ed> 
 
-Now average the two responses and that will be your score for this capability.
+<ed> I suggest adding a line or two here that clarifies your vision for next steps for readers based on their score. For example, if their score was a 4, they're good to go here and can opt to move on to Related Capabilities? If their score was 3 or below, they should see Supporting Practices? </ed>
 
 ## Supporting Practices
 
+The following supporting practices will bolster your team’s Code Maintainability capability.
```

Comment:
What about something less definitive here that encourages consideration and experimentation? We want to be consistent with the text in the main README.md file which is as follows:

> Not every practice we include will be beneficial in every situation. Each unique situation is rife with nuance. Our curated list is designed to include practices that typically work and are in support a given DORA Capability. Our goal is to ensure that teams always have fresh, pragmatic, and actionable ideas on how they can improve.

Reply:
What about: "The following is a curated list of supporting practices to consider when looking to improve your team's Code Maintainability capability. While not every practice will be beneficial in every situation, they are meant to provide teams with fresh, pragmatic, and actionable ideas to support this capability."

Reply:
Perfect

Reply:
Edited.